package.json

```diff
@@ -97,7 +97,7 @@
     "icon": "https://user-images.githubusercontent.com/321738/63501763-88dbf600-c4cc-11e9-96cd-94adadc2fd72.png"
   },
   "dependencies": {
-    "@storybook/csf": "0.0.2--canary.87bc651.0",
+    "@storybook/csf": "0.0.2-alpha.0",
```

Comment:
I think we should use the same canary as the storybook monorepo. I promise I'll clean this up later this week

Reply:
done